docs/101/allocations.md

```diff
@@ -26,23 +26,73 @@ accounts balance
 ```
 
 ## How do I review the usage of my allocation?
-The `accounts` tool provides several options for summarizing allocation usage. To obtain a summary, execute the command:
+The `accounts` tool provides several options for summarizing the usage of the allocations to which your RCC account is accessible. To obtain a summary, execute the command `accounts usage` with the account name(s):
+
+
+=== "Midway2, DaLI"
+      ```
+      accounts usage
+      accounts usage --account [pi-cnetid]
+      accounts usage --accounts [pi-cnetid1],[pi-cnetid2]
+      ```
+      
+===+ "Midway3, Beagle3"
+      ```
+      accounts usage
+      accounts usage -a [pi-cnetid]
+      ```
+
+where `[pi-cnetid]` is the name of the account, often your PI's.
+
+You may use the `--byuser` option to view individual usage by group members:
+
+=== "Midway2, DaLI"
+      ```
+      accounts usage
+      accounts usage --account [pi-cnetid] --byuser
+      ```
+
+===+ "Midway3, Beagle3"
+      ```
+      accounts usage
+      accounts usage -a [pi-cnetid] -byuser
+      ```
+
+You can also use the `rcchelp` command (a wrapper of system-provided tools including `accounts`), which requires slightly different arguments (note only one character '-')
+
+=== "Midway2, DaLI"
+      ```
+      rcchelp usage 
+      rcchelp usage -account [pi-cnetid] -byuser
+      ```
+
+=== "Midway3, Beagle3"
+      ```
+      rcchelp usage 
+      rcchelp usage -account [pi-cnetid] -byuser
+      ```
 
-```
-accounts usage
-```
+---
 
-To view usage per job, utilize the following command:
+To view the resource usage per job, you need to run on Midway2 before using either `accounts`:
 
-```
-accounts usage --byjob
-```
+=== "Midway2, DaLI"
+      ```
+      accounts usage --byjob
+      accounts usage --account [pi-cnetid] --byjob
+      ```
 
-If you are the PI, you may employ the `--byuser` option to view individual usage by group members:
+or `rcchelp`
 
-```
-accounts usage --byuser
-```
+=== "Midway2, DaLI"
+      ```
+      rcchelp usage 
+      rcchelp usage -account [pi-cnetid] -byjob
+      ```
+
+both of which include the jobs on both Midway2 and Midway3 partitions (e.g. `caslake` and `gpu`). 
+
+The output is a table that lists the job IDs, partitions, number of CPU cores and SU consumptions. To further check the resource consumption of a job on Midway3, you need to log in to Midway3 and run `sacct -j` or `scontrol show job` with the job ID. 
 
```

Comment:
maybe to note that `sacct` will work during and after the execution. but, `scontrol` will work only during, and not after.